app/models/post.rb

```diff
@@ -0,0 +1,9 @@
+class Post < ApplicationRecord
+  belongs_to :user, counter_cache: :posts_counter
+  has_many :comments, dependent: :destroy
+  has_many :likes, dependent: :destroy
+
+  def recent_comments(limit = 5)
+    comments.last(limit)
+  end
+end
```

Comment:
- [ ] Post model should a method that updates the posts counter for a user so would you add this method as well?

Reply:
Well the counters update automatically thanks to the Rails counter_cache. They say we can improve on things that we believe is better.

```rb
  belongs_to :user, counter_cache: :posts_counter
```

Reply:
Try creating and deleting models in Rails console and see it in effect

Reply:
@de-scholar